### DISCUSSION TITLE
How to start training?

### DISCUSSION THREAD

**Discussion by ashleycolton** [Equim-chan/Mortal#62] [Q&A]
I don't find any instructions in the readme.md 
tried to cargo run but    
Compiling libriichi v0.1.0 (C:\Users\Admin\Desktop\Mortal\libriichi)
    Finished dev [unoptimized + debuginfo] target(s) in 14.06s
     Running `target\debug\stat.exe`
Error: Usage: stat <DIR> <PLAYER_NAME>
error: process didn't exit successfully: `target\debug\stat.exe` (exit code: 1)

also running mortal.py will get 
ModuleNotFoundError: No module named 'libriichi'
ERROR: No matching distribution found for libriichi

**Comment by hyskylord** [comment]
There is a URL to the documentation in readme.
